src/ui/AutoCompleteSuggest.ts

```diff
@@ -30,14 +30,14 @@ function suggestWords(words: Word[], query: string, max: number): Word[] {
   const queryStartWithUpper = capitalizeFirstLetter(query) === query;
   return Array.from(words)
     .map((x) => {
-      if (x.value === query) {
+      if (x.value === query || x.value.length < query.length) {
```

Comment:
@chenyukang 
Does this relate to this issue?

It will improve performance. On the other hand, there will be some problems when aliases (Custom dictionary and internal link) are longer than value. So I want to solve it after releasing v3.0.0.

Reply:
> 

I rollback this change now.